src/ai/susi/server/api/aaa/CreateGroups.java

```diff
@@ -0,0 +1,80 @@
+/**
+ *  CreateGroups
+ *  Copyright 22.06.2017 by saurabh
+ *
+ *  This library is free software; you can redistribute it and/or
+ *  modify it under the terms of the GNU Lesser General Public
+ *  License as published by the Free Software Foundation; either
+ *  version 2.1 of the License, or (at your option) any later version.
+ *
+ *  This library is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *  Lesser General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Lesser General Public License
+ *  along with this program in the file lgpl21.txt
+ *  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package ai.susi.server.api.aaa;
+
+import ai.susi.DAO;
+import ai.susi.json.JsonObjectWithDefault;
+import ai.susi.server.*;
+import org.json.JSONObject;
+
+import javax.servlet.http.HttpServletResponse;
+import java.io.File;
+
+/**
+ * This Servlets creates a group with default baserole as admin.
+ * It takes 1 parameter group and other optional parameter role.
+ * Can be tested on
+ * http://127.0.0.1:4000/aaa/createGroups.json?group=groupName&role=admin
+ *
+ */
+public class CreateGroups extends AbstractAPIHandler implements APIHandler {
+
+
+    private static final long serialVersionUID = -742269505564698987L;
+
+    @Override
+    public BaseUserRole getMinimalBaseUserRole() {
+        return BaseUserRole.ADMIN;
+    }
+
+    @Override
+    public JSONObject getDefaultPermissions(BaseUserRole baseUserRole) {
+        return null;
+    }
+
+    @Override
+    public String getAPIPath() {
+        return "/cms/createGroups.json";
```

Comment:
This servlet is in the aaa Package and therefore the path should be "/aaa/createGroup.json";

not "groups". It creates one group. Therefore "group"